infra/aws/terraform/kops-infra-ci/eks.tf

```diff
@@ -99,7 +99,7 @@ module "eks" {
       }
 
       capacity_type  = "ON_DEMAND"
-      instance_types = ["r7i.2xlarge"]
+      instance_types = ["r6id.2xlarge"]
```

Comment:
we need to mount and expose the instance store volumes before they can be utilized (and accounted for by kubelet's calculation of ephemeral storage) - here is how we can do that https://github.com/awslabs/data-on-eks/blob/9cde4cf821d21eefea58f83ade62f0cf476d8826/analytics/terraform/spark-k8s-operator/main.tf#L91-L104

Reply:
interesting! thanks for the tip @bryantbiggs

Reply:
@bryantbiggs Added.

Reply:
related PR for the curious https://github.com/awslabs/amazon-eks-ami/pull/1171